zer0-os/zOS
# Real-time Notifications

### What does this do?

Part 1 introduces real-time notifications using Pusher to connect to our pre-existing Pusher events from the API to our Notifications. Part 2 will introduce browser events for these notifications.

### How do I test this?

- As User A open zOS, click on the Notifications to open the modal
- As User B open Zero Web
- As User B mention User A in a message
- As User A verify that a new notification has arrived in your Notifications modal
